### PR TITLE
bridge_between_zulips: Add explanation of --stream to README.md.

### PR DESCRIPTION
**Pull request by rht**

### CODE REVIEW

File: zulip/integrations/bridge_between_zulips/run-interrealm-bridge [hunk 1]
```diff
@@ -64,8 +64,10 @@ if __name__ == "__main__":
     Relay each message received at a specified subject in a specified stream from
     the first realm to a specified subject in a specified stream in the second realm.
 
-    If the --stream flag is added, then the tunnel granularity becomes
-    stream-wide. All subjects are mirrored as-is without translation.
+    If the --stream flag is added, then the mirroring
+    granularity becomes stream-wide. Instead of one subject,
+    all subjects within the stream are mirrored as-is without
+    translation.
```
Comment (timabbott) [inline zulip/integrations/bridge_between_zulips/run-interrealm-bridge]: s/subject/topic/, right?

Comment (rht) [reply zulip/integrations/bridge_between_zulips/run-interrealm-bridge]: Fixed in the 2nd commit.

**Comment by rht** [comment]
@timabbott ?

**Comment by timabbott** [comment]
Merged, thanks @rht!

**Pull request merged**